lbjava/pom.xml

```diff
@@ -29,7 +29,6 @@
 			<groupId>gurobi</groupId>
 			<artifactId>gurobi</artifactId>
 			<version>5.0.1</version>
-			<optional>true</optional>
```

Comment:
I vote to keep this optional flag, to avoid pushing people the wrong version of Gurobi. They should import whatever they have a licence for.